### ISSUE TITLE
sort likes list

### ISSUE THREAD

**Issue by andrewjs18** [MyBBStuff/SimpleLikes#22]
it'd be a nice enhancement if we can turn on sorting for the likes list in the postbit so it sorts the list via username from asc-desc, etc., assuming this doesn't add too much load.

**Comment by euantorano** [comment]
I can't remember how it's currently sorted. Perhaps it should sort by time and username (eg: most recent first, and if two people liked at the same time, sort their names alphabetically)?

**Comment by andrewjs18** [comment]
yeah, I think that'd be ideal, although now that I'm looking at it, I think that's how it already is sorted.

when I initially wrote this 'issue', I was still wanting to try to mimic the collapsible list from the thank you/like plugin.  after seeing the way this plugin, along with most other like features/plugins from other forum softwares is laid out, this seems fine.

I guess I can leave this open so if you want to add a setting to allow the user to sort it in a specific manner that suits their needs, then that'd be fine.
